packages/dds/merge-tree/src/collections/list.ts

```diff
@@ -33,11 +33,12 @@ class HeadNode<T> {
 	public get prev(): DataNode<T> | undefined {
 		return this._prev === this.headNode ? undefined : (this._prev as DataNode<T>);
 	}
-	public get list() {
+	public get list(): DoublyLinkedList<T> | undefined {
 		return this.headNode._list;
 	}
 }
 
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
Any reason we can't convert this to a real type? If so, let's leave a comment.

Reply:
And if you can't get a real type, then `unknown` is better than `any`. Can't use `unknown` either?

Reply:
I don't think I can use `unknown`, at least not without modifying all of the nodes to be of type `T | unknown`.